src/game.cpp

```diff
@@ -3725,7 +3725,8 @@ std::vector<std::string> game::list_active_characters()
  */
 void game::write_memorial_file(std::string sLastWords)
 {
-    const std::string &memorial_dir = FILENAMES["memorialdir"];
+    const std::string &memorial_dir = FILENAMES["memorialdir"] +  world_generator->active_world->world_name + "/";
```

Comment:
If it fails, it fails with a debug message, not a crash. (That is what happened in earlier testing when I used world_path, which appended to much to the path). So there is at least that. Sadly I don't have access to a win32 machine. We could ask around on reddit or something.